autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -238,7 +254,15 @@ def update_execution(status: ExecutionStatus) -> NodeExecutionResult:
 
     except Exception as e:
         error_msg = str(e)
-        db_client.upsert_execution_output(node_exec_id, "error", error_msg)
```

Comment:
I know it's not your change, but would you mind removing `# Update execution status and spend credits` comment above in this PR? :)))